doc/source/security/using-keyring-plugin.rst

```diff
@@ -0,0 +1,69 @@
+.. using-keyring-plugin:
+
+=======================================================
+Using the Keyring Plugin
```

Comment:
Why is there a keyring.rst (describing basically the keyring plugins) too?

Reply:
In the process of rearranging the info from keyring.rst before I delete it.

Reply:
keyring.rst is deleted.